metadata-ingestion/README.md

```diff
@@ -343,6 +344,29 @@ source:
     # table_pattern/schema_pattern is same as above
 ```
 
+### Druid `druid`
+
+Extracts:
+
+- List of databases, schema, and tables
+- Column types associated with each table
+
+**Note** It is important to define a explicitly define deny schema pattern for internal druid databases (lookup & sys) 
+if adding a schema pattern otherwise the crawler may crash before processing relevant databases.
+This deny pattern is defined by default but is overriden by user-submitted configurations
+
+```yml
+source:
+  type: druid
+  config:
+    # Point to broker address
+    host_port: localhost:8082
+    schema_pattern:
+      deny:
+        - "^(lookup|sys).*"
```

Comment:
I wonder if we should "hardcode" this inside the Druid config class so this is always added on regardless of what the user configures.